### PR TITLE
Sprint IC integration

### PR DESCRIPTION
**Pull request by pmumbere**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RET-2025
https://tools.hmcts.net/jira/browse/RET-2027
https://tools.hmcts.net/jira/browse/RET-2028
https://tools.hmcts.net/jira/browse/RET-2029
https://tools.hmcts.net/jira/browse/RET-2037
https://tools.hmcts.net/jira/browse/RET-2045
https://tools.hmcts.net/jira/browse/RET-2047
https://tools.hmcts.net/jira/browse/RET-2026

### Change description ###
Initial consideration Rule 26 pages

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: definitions/json/CaseEventToFields.json [hunk 1]
```diff
@@ -2936,6 +2936,7 @@
     "PageID": 10,
     "PageDisplayOrder": 10,
     "PageFieldDisplayOrder": 14
+
```
Comment (Itay-Kirsh) [inline definitions/json/CaseEventToFields.json:2939]: What are these line breaks for?

File: definitions/json/CaseEventToFields.json [hunk 2]
```diff
@@ -3525,6 +3528,416 @@
     "ShowSummaryChangeOption": "Y",
     "PageColumnNumber": 1
   },
+  {
+    "CaseTypeID": "ET_Scotland",
+    "CaseEventID": "initialConsideration",
+    "CaseFieldID": "etInitialConsiderationRespondent",
```
Comment (Itay-Kirsh) [inline definitions/json/CaseEventToFields.json:3534]: Change etInitialConsideration to etIC for the first few fields as this is the convention in the rest of the commit. With Ctrl+F found 61 cases of etInitialConsideration and 310 of etIC.

**Review by Itay-Kirsh — CHANGES_REQUESTED** [review]

File: definitions/json/CaseEventToFields.json (hunk 1 above)
Comment (pmumbere) [reply definitions/json/CaseEventToFields.json]: I think happened post merge. Now resolved

File: definitions/json/CaseEventToFields.json (hunk 2 above)
Comment (pmumbere) [reply definitions/json/CaseEventToFields.json:3532]: We are aware. However, we have already made changes to data model and callbacks etc. We will carry out a refactoring exercise later on

**Review by Itay-Kirsh — APPROVED** [review]

**Review by hsjhita — APPROVED** [review]

File: definitions/json/ComplexTypes.json [hunk 3]
```diff
@@ -4957,5 +4957,284 @@
...
+    "FieldType": "Text",
+    "FieldShowCondition": "etICRule27ClaimToBe CONTAINS \"dismissedInPart\"",
+    "SecurityClassification": "Public",
+    "RetainHiddenValue": "No"
+  },
+  {
+    "ID": "etInitialConsiderationRule27",
+    "ListElementCode": "etICRule27Direction",
+    "ElementLabel": "Employment Judge's direction for Rule 27 Notice",
+    "FieldType": "MultiSelectList",
```
Comment (hsjhita) [inline definitions/json/ComplexTypes.json:5184]: FixedRadioList according to Sigma

File: definitions/json/ComplexTypes.json [hunk 4]
```diff
@@ -4957,5 +4957,284 @@
...
+    "ListElementCode": "etICRule27NumberOfDays",
+    "ElementLabel": "Number of days for claimant to provide written representations",
+    "FieldType": "Number",
+    "SecurityClassification": "Public",
+    "RetainHiddenValue": "No"
+  },
+  {
+    "ID": "etInitialConsiderationRule28",
+    "ListElementCode": "etICRule28ClaimToBe",
+    "FieldType": "MultiSelectList",
```
Comment (hsjhita) [inline definitions/json/ComplexTypes.json:5209]: FixedRadioList according to Figma

File: definitions/json/ComplexTypes.json (hunk 3 above)
Comment (pmumbere) [reply definitions/json/ComplexTypes.json:5184]: Fixed

File: definitions/json/ComplexTypes.json (hunk 4 above)
Comment (pmumbere) [reply definitions/json/ComplexTypes.json]: Fixed

**Review by Itay-Kirsh — APPROVED** [review]

File: definitions/json/AuthorisationCaseField.json [hunk 5]
```diff
@@ -10181,6 +10181,1170 @@
...
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "etICConvertF2fGiveDetails",
+    "UserRole": "caseworker-employment-scotland",
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "etICConvertF2fGiveDetails",
+    "UserRole": "caseworker-employment",
+    "CRUD": "R"
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField.json:11346]: missing auth for caseworker-employment-api

Comment (pmumbere) [reply definitions/json/AuthorisationCaseField.json:11346]: resolved

**Review by Itay-Kirsh — DISMISSED** [review]

**Review by Itay-Kirsh — APPROVED** [review]

**Pull request merged**